webapi/src/test/scala/org/knora/webapi/util/search/v2/SearchParserV2Spec.scala

```diff
@@ -293,6 +293,34 @@ class SearchParserV2Spec extends CoreSpec() {
                 operator = CompareExpressionOperator.LESS_THAN,
                 leftArg = QueryVariable("pubdate")
             ))
+        ), positiveEntities = Set(
+            IriRef("http://0.0.0.0:3333/ontology/incunabula/simple/v2#publoc".toSmartIri, None),
+            XsdLiteral("14", "http://www.w3.org/2001/XMLSchema#integer".toSmartIri),
```

Comment:
No reason to have an `XsdLiteral` here, see comment above.

Reply:
right